DOCS-1258: Make ML training API page (#2164)
viamrobotics/docs: docs/program/apis/_index.md

{{< readfile "/static/include/services/apis/data-client.md" >}}

### ML Training API

> The ML training API allows you to get data from and cancel ML training jobs taking place on the [Viam app](https://app.viam.com):

Reviewer: ```suggestion
The ML training API allows you to get information about and cancel ML training jobs taking place on the [Viam app](https://app.viam.com):
```
